public/traffic_monitoring_graphs.js

```diff
@@ -0,0 +1,1294 @@
+// GRAPH CONTROLLER
+class GraphController {
```

Comment:
Am I right this has just moved from somewhere else? If so, does it need removing from that old place?

Reply:
yeah, I'll remove it, if you delete graph.js file, it leads to conflict with master which can't be resolved.